src/index.html

```diff
@@ -1,13 +1,51 @@
 <!doctype html>
 <html lang="en">
   <head>
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article class="card" data-qa="card">
+      <h1 class="product-name">
+        APPLE A1419 iMac 27" Retina <br>
+        5K Monoblock (MNED2UA/A)
+      </h1>
+
+      <p class="product-code">
+        Product code: 195434
+      </p>
+
+      <p class="review">Reviews: 5</p>
+
+      <div class="stars">
```

Comment:
<img width="851" alt="Screenshot 2023-08-09 at 18 02 12" src="https://github.com/mate-academy/layout_product-cards/assets/53228259/5325c83d-82b0-4006-b90d-771b13e6b833">

Reply:
This is not resolved here should be a modifier